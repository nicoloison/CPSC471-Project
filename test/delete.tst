curl localhost/recipes/delete.php
curl "localhost/recipes/delete.php?recipe_name=ribs"
curl "localhost/recipes/delete.php?recipe_name=ribs&author_name=adjuric"