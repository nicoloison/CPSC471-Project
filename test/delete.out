{"result":"failure","error":"missing parameter recipe_name"}{"result":"failure","error":"missing parameter author_name"}{"result":"success","error":""}